maps-compose/src/main/java/com/google/maps/android/compose/Marker.kt

```diff
@@ -148,10 +114,10 @@ public fun rememberMarkerState(
 @Composable
 @GoogleMapComposable
 public fun Marker(
+    position: LatLng,
```

Comment:
This is definitely the Compose way of representing a marker's position. However, since this is an interop library, the issue with this design is that the marker's position is owned by the Maps SDK. So when you drag the marker to a new location, this value is no longer correct. This is why `MarkerState` was introduced and needed as a workaround to the internal behavior of the Maps SDK.

Reply:
Please see my comments in #149 regarding how to keep marker position up-to-date.

Reply:
Moving `position` here would definitely match the mental model of Compose though I think changing the behavior of what the Maps SDK is doing under the hood to create a better Compose API might create more problems down the line. 

I see your comment below about reworking a solution. My 2c of a summary of proposed changes that would solve your particular use case is:
* Add `onDrag: (LatLng, DragEvent) ->` lambda to `Marker`
* Deprecate `MarkerState.dragState`
* Add `MarkerState.isDragging`